stdlib/gc.mli

```diff
@@ -300,34 +305,35 @@ external compact : unit -> unit = "caml_gc_compaction"
 
 val print_stat : out_channel -> unit
 (** Print the current values of the memory management counters (in
-   human-readable form) into the channel argument. *)
+   human-readable form) of the total program into the channel argument. *)
 
 val allocated_bytes : unit -> float
-(** Return the total number of bytes allocated since the program was
-   started.  It is returned as a [float] to avoid overflow problems
+(** Return the number of bytes allocated by this domain and potentially
+   a previous domain. It is returned as a [float] to avoid overflow problems
    with [int] on 32-bit machines. *)
 
 external get_minor_free : unit -> int = "caml_get_minor_free"
-(** Return the current size of the free space inside the minor heap.
+(** Return the current size of the free space inside the minor heap of this
+   domain.
 
     @since 4.03.0 *)
 
-external get_bucket : int -> int = "caml_get_major_bucket" [@@noalloc]
+(*external get_bucket : int -> int = "caml_get_major_bucket" [@@noalloc] *)
 (** [get_bucket n] returns the current size of the [n]-th future bucket
```

Comment:
I wonder where this comment will be attached by `ocamldoc`. Either remove the function entirely, or comment out the function declaration **and** the ocamldoc comment altogether. Same for the next two functions.

Reply:
Fixed in 5aa07b2